helm/values.yaml

```diff
@@ -2,11 +2,11 @@ environment: development
 postgresUsername: postgres@mine-support2
 postgresPassword: changeme
 postgresDatabase: mine_payments
```

Comment:
Postgres database still has `mine` in the name. I think this one is ok as that's a fair description of what the database contains. Just mentioning it in case I'm wrong as it will be just about the only mention of "mine" in the project.

Reply:
There's a different card to rename the databases.  Involves changing not only the names of the database but the server in Azure hosting them too.  This can be addressed as part of that story.